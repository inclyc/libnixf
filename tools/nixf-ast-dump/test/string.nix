# RUN: nixf-ast-dump < %s | FileCheck %S/string.expected
"asdasdasd${ "string" }asdasd"
