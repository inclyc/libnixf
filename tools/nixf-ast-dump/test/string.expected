CHECK:      String
CHECK:      Token 9 asdasdasd
CHECK-NEXT: Interpolation 11
CHECK-NEXT:  Token 2 ${
CHECK-NEXT:  String 8
CHECK-NEXT:   Token 1  "
CHECK-NEXT:   Token 6 string
CHECK-NEXT:   Token 1 "
CHECK-NEXT:  Token 1  }
CHECK-NEXT: Token 6 asdasd
CHECK-NEXT: Token 1 "
